src/ontology/mondo-edit.obo

```diff
@@ -400975,19 +400959,37 @@ intersection_of: disease_has_location UBERON:0009853 ! body of uterus
 id: MONDO:0021651
 name: synpolydactyly
 def: "A joint presentation of syndactyly (fusion of digits) and polydactyly (production of supernumerary digits)." [https://en.wikipedia.org/wiki/Synpolydactyly]
+subset: gard_rare
+subset: ordo_morphological_anomaly {source="Orphanet:93403"}
 synonym: "polysyndactyly" EXACT [NCIT:C125597, SCTID:84598000, UMLS:C0265553]
+synonym: "SPD" RELATED ABBREVIATION [GARD:0005087]
+synonym: "syndactyly type 2" EXACT [MONDO:0019683]
+synonym: "synpolydactyly" EXACT [Orphanet:93403]
 synonym: "synpolydactyly" EXACT [NCIT:C75003]
+xref: GARD:0005087 {source="MONDO:equivalentTo"}
+xref: ICD10:Q70.0 {source="ORDO:93403/nd", source="ORDO:93403/attributed", source="Orphanet:93403"}
+xref: ICD10:Q70.2 {source="ORDO:93403/nd", source="ORDO:93403/attributed", source="Orphanet:93403"}
+xref: MESH:C538153 {source="MONDO:subClassOf", source="ORDO:93403/e", source="Orphanet:93403"}
 xref: MESH:C538153 {source="MONDO:equivalentTo"}
```

Comment:
this xref is duplicated, but have MONDO:equivalentTo and MONDO:subClassOf

Reply:
oh thanks! I'll fix that